lib/lynx/config.rb

```diff
@@ -9,26 +9,25 @@ def initialize(config = {})
       @config = config
     end
 
-    [:username, :password, :host, :database, :socket].each do |method|
+    [:credentials, :username, :host, :database, :socket].each do |method|
       define_method(method){ self[method] }
     end
 
     def mysql
-      @mysql ||= self[:mysql] || detect(MYSQL)
+      @mysql ||= self[:mysql] || detect(MYSQL) || SystemOut.system_out_with_err('[ERROR] Failed to detect a valid version of mysql')
```

Comment:
How does this work? I thought this method expected a command.

Reply:
This was a brain fart